src/main/java/com/ksm/wstbackoffice/entity/City.java

```diff
@@ -0,0 +1,12 @@
+package com.ksm.wstbackoffice.entity;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@EqualsAndHashCode
```

Comment:
Please use Lombok annotation @ Data instead of multiple annotations you are using (and change in all other classes).
Here you can read more about Data annotation: https://projectlombok.org/features/Data

Reply:
done